mailet/standard/src/main/java/org/apache/james/transport/mailets/AmqpForwardAttribute.java

```diff
@@ -106,12 +110,11 @@ public void init() throws MailetException {
 
     @Override
     public void service(Mail mail) throws MailetException {
-        if (mail.getAttribute(attribute) == null) {
-            return;
-        }
-        Stream<byte[]> content = getAttributeContent(mail);
         try {
-            sendContent(content);
+            Optional<Attribute> fetchedAttribute = mail.getAttribute(attribute);
+            if (fetchedAttribute.isPresent()) {
```

Comment:
I smell a bad Optional use that could be solved by:
 - Using `ifPresent`
 - Handling all exceptions in `sendContent` so that it no more returns checked exceptions